sql/postgres/sqlspec.go

```diff
@@ -257,12 +257,20 @@ func columnTypeSpec(t schema.Type) (*sqlspec.Column, error) {
 		return &sqlspec.Column{TypeName: t.T}, nil
 	case *schema.UnsupportedType:
 		return &sqlspec.Column{TypeName: t.T}, nil
-	case *NetworkType:
-		return &sqlspec.Column{TypeName: t.T}, nil
-	case *CurrencyType:
+	case *ArrayType:
 		return &sqlspec.Column{TypeName: t.T}, nil
 	case *BitType:
 		return bitSpec(t)
+	case *CurrencyType:
+		return &sqlspec.Column{TypeName: t.T}, nil
+	case *NetworkType:
+		return &sqlspec.Column{TypeName: t.T}, nil
+	case *SerialType:
+		return &sqlspec.Column{TypeName: t.T}, nil
+	case *UUIDType:
+		return &sqlspec.Column{TypeName: t.T}, nil
+	case *XMLType:
+		return &sqlspec.Column{TypeName: t.T}, nil
 	default:
 		return nil, fmt.Errorf("failed to convert column type %T to spec", t)
```

Comment:
Went over this with @yonidavidson on the MySQL package a little while ago, I think the correct approach is:
```suggestion
	default:
		return &sqlspec.Column{TypeName: t.T}, nil
```

Reply:
And same goes for all that types that this function is merely a pass-through for `&sqlspec.Column{TypeName: t.T}, nil`

Reply:
A type assertion is required for getting `.T`.

Reply:
got it 👍